libs/portals/admin/ids-admin/src/components/Applications/Applications.tsx

```diff
@@ -44,7 +43,7 @@ const Applications = () => {
             <GridRow key={item.id}>
               <Link
                 className={styles.fill}
-                to={`/innskraningarkerfi/${tenant}/forrit/${item.id}/`}
+                to={`/innskraningarkerfi/${tenant}/forrit/${item.id}`}
```

Comment:
~Wondering if we should use `IDSAdminPaths` and use your new `replaceParams` method to prevent broken urls if we change `IDSAdminPaths`